packages/sdk/src/types/Loyalties.ts

```diff
@@ -0,0 +1,500 @@
+import { VouchersResponse } from './Vouchers'
+import { ValidationRulesCreateAssignmentResponse } from './ValidationRules'
+import { SimpleCustomer } from './Customers'
+import { OrdersItem } from './Orders'
+import { ProductsCreateResponse, ProductsCreateSkuResponse } from './Products'
+
+interface LoyaltiesVoucher {
+	code_config?: {
+		length?: number
+		charset?: string
+		pattern?: string
+		prefix?: string
+		suffix?: string
+	}
+	type?: string
+	is_referral_code?: boolean
+	loyalty_card?: {
+		points: number
+		balance: number
+	}
+	redemption?: {
+		quantity?: number
+	}
+}
+
+export interface LoyaltiesListParams {
+	limit?: number
+	page?: number
+}
+
+export interface LoyaltiesListResponse {
+	object: 'list'
+	total: number
+	data_ref: 'campaigns'
+	campaigns?: LoyaltiesCreateCampaignResponse[]
+}
+
+export interface LoyaltiesCreateCampaign {
+	name: string
+	start_date?: string
+	expiration_date?: string
+	type?: 'AUTO_UPDATE' | 'STATIC'
+	vouchers_count?: number
+	voucher: {
+		type?: 'LOYALTY_CARD'
```

Comment:
Drop `?` - it's loyalty campaign creation then not having this field would start returning 400 from server that `discount` property is required which do not make sense for `loyalty`. In other words trying to create loyalty campaign without that field set to `LOYALTY_CARD` is a nonsense.